vrk-kpa/suomidigi
# SUOPA-138 Sidebar

To test: 
- `make fresh`
- Go to http://suomidigi.fi.docker.amazee.io and see that there's a sidebar
   - The sidebar should be display: none at mobile width
- Check that the font sizes and line heights matches these: https://vrk-ewiki.eden.csc.fi/display/SFIYHT/Typografia
